### ISSUE TITLE
Immediately dying when starting a game.

### ISSUE THREAD

**Issue by Rencer** [Unarelith/SkyFactory#3]
I created a new game and when I staring it I almost immediately die.

For a few seconds I can look around (nothing just a void and some clouds in the sky) and the only thing I can see is my character -with a book in its hand- is falling down into the void and then dies.
I tried with every mapgen, but it is always the same; falling for few seconds and then I die.
If I click on "respawn" than I spawn in an island with my fresh bones.

Is this a normal behaviour?

**Comment by Unarelith** [comment]
Yes, this bug sometimes happened when I was developing this game, I don't know for sure why and how, but I'll find out once I restart the development.

**Comment by Rencer** [comment]
Thanks, good to hear that you still working on this game, I really like anything that is skyblock.
I like that you even make the mods that designed to work in skyblock game.
I find out that if I grab my bones, I can turn them into bonemeal, so at least we get something when we die without a reason.

**Comment by Unarelith** [comment]
This issue is still a thing so I'll keep this open until I really resolve it

**Comment by programmerjake** [comment]
@Rencer please change the title back since just a period is not a description of the bug

**Comment by Rencer** [comment]
> @Rencer please change the title back since just a period is not a description of the bug

Are you happy now?
This fucking fucktard of who make this shit is FUCKING STALKERS! AND I DON'T WANT TO HEAR ANYTHING FROM THEM!
But this fucking GitFuckHUb still SPAMMING MY ACTIVITY Whit THIS SHIT!
I was the only one whoever posted lot of COOL suggestion ato make this shit mod better, but NO!
This FUCKTARD ASHHOLE said that I'M ían idiot.
Not to mewntion thsi FUCKING FUCKHEAD is AGAINST EVERYTHING EVEN IN THE OFFICAL MINETEST FORUM, AGAINST EVERYTHING THAT POSSIBLY MAKE THE GAMNE BETTER!
Everything that requested and APPROVED by the makers of Minetst is't get implemented to the game, BECAUSE OF THIS FUCKTARD MODDER! DIE YOU MOTHERFUCKER!!!

**Comment by Rencer** [comment]
LOT OF TYPO BECAUS E I ANGRY!
get the fuck out of MINETEST you piece of shit!

**Comment by Unarelith** [comment]
Uh? Wth are you talking about?
